Script/PRONTO.py

```diff
@@ -457,7 +458,12 @@ def get_patient_info_from_MTF_new(ipd_material_file,ipd_no,DNA_sampleID,RNA_samp
 					datetime_date = str(xlrd.xldate_as_datetime(ipd_birth_date_exl,0))
 					ipd_birth_year = datetime_date.split('-')[0]
 				except:
-					ipd_birth_year = "-"
+					try:
+						day, month, year = map(int, ipd_birth_date_exl.split('.'))
```

Comment:
does this work?
```suggestion
						birthdate = datetime.strptime(ipd_birth_date_exl, '%d.%m.%Y').date()
						ipd_birth_year = str(birthdate.year)
```

Reply:
could it be that `ipd_birth_date_exl` is not a string? You can keep your change in any case. Was just wondering if that was a solution.